### ISSUE TITLE
MC crash

### ISSUE THREAD

**Issue by notIkibo** [DakotaPride/More-Weaponry-1.18#12]
[crash-2022-11-07_18.53.26-client.txt](https://github.com/DakotaPride/More-Weaponry/files/9954254/crash-2022-11-07_18.53.26-client.txt)
I guess my mc crashed due to errors, provided by moreweaponry

**Comment by DakotaPride** [comment]
Is on Fabric or Quilt? As this is typically a common error for Quilt users. Also, is this on a modpack (whether it be custom or an established one)? Again, pretty common error for users who are making/using a quilt modpack.

**Comment by notIkibo** [comment]
It's Fabric :(

**Comment by DakotaPride** [comment]
And is it a modpack? Compatibility issues can happen depending on the case. You should also provide whether or not this is on startup or if it happens at some point while actually playing.

**Comment by notIkibo** [comment]
It is a modpack and it happens on startup. Though as I have disabled moreweaponry, I could start MC, but everytime I tried to play a Singleplayer world I get disconnected with the message "No valid player data" (Note: I played on Fabric 0.14.9 before and no, I don't play cracked mc) so maybe it's either I have more than one issue or due to my unknown rootproblem More Weaponry also gets to its knees

**Comment by DakotaPride** [comment]
I think I may have found the issue, let me know if I'm wrong though. The version that you're using (v2.2.5) uses Fabric API version 0.60.0 for 1.19.2, while you're currently using 0.60.0. I recommend updating More Weaponry to the latest version, which uses 0.64.0, however, if your mods limit you to loader version 1.14.9 (the most recent More Weaponry version uses loader 1.14.10), then I might suggest removing More Weaponry until you are no longer limited by this. Thank you for your patience with this issue and let me know if you have any more issues/concerns!

**Comment by DakotaPride** [comment]
Side question: Is the mod 'Deeper And Darker' in the modpack you are using? If so, remove and try again, as it seems to be incompatible with my other mod 'Boleatte', so most likely will be incompatible with 'More Wepaonry' as well.

**Comment by notIkibo** [comment]
Alright soooo: I used 0.64.0 (and now there is 0.66.0) so thats not the case, buuuut I do have DAD in my modpack, but not in my life (I had to). So I guess that answers my issue. Though I would think that since Boleatte has its own dimension and stuff (I really have no clue of modding), it maybe is incompatible in that sense? Unless Weaponry and Boleatte change code in the same areas (which (not my) DAD also does). 

Btw I need Boleatte to be more popular for Aternos to accept the mod. I like harsh air 🥶

Oh and curseforge is shitty rn and my "no valid data" comes from auth servers being down. 
![image](https://user-images.githubusercontent.com/106886638/200492237-02be615a-bdb6-4273-a8a1-54b20df8ab97.png)
